Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -130,6 +130,22 @@ typedef int32_t   hsError;
 #   define hsDeprecated(message)
 #endif
 
+#ifdef HAVE_OVERRIDE
+#   define HS_OVERRIDE  override
```

Comment:
We've specified our minimum MSVC compiler, but I don't know what we're targetting for GCC now that it has better support.  IIRC, GCC only added override/final in version 4.7